bin/cml.test.js

```diff
@@ -31,3 +32,21 @@ describe('command-line interface tests', () => {
     `);
   });
 });
+
+describe('Valid Docs URLs', () => {
+  test.each([
+    'workflow/rerun',
+    'tensorboard/connect',
+    'runner/launch',
+    'repo/prepare',
+    'pr/create',
+    'comment/create',
+    'comment/update',
+    'check/create',
+    'asset/publish'
```

Comment:
```suggestion
    'workflow:rerun',
    'tensorboard:connect',
    'runner:launch',
    'repo:prepare',
    'pr:create',
    'comment:create',
    'comment:update',
    'check:create',
    'asset:publish'
```

For consistency with analytics?

Reply:
My bad, nevermind

Reply:
going to be hard to keep this list updated as (sub)commands new are added.